packages/databricks-vscode/src/configuration/ConnectionManager.ts

```diff
@@ -256,227 +239,81 @@ export class ConnectionManager {
             await this.waitForConnect();
         }
 
-        this._projectConfigFile = undefined;
         this._workspaceClient = undefined;
         this._databricksWorkspace = undefined;
-        this.updateCluster(undefined);
-        this.updateSyncDestination(undefined);
+        await this.setClusterManager();
+        await this.setSyncDestinationMapper();
```

Comment:
Nitpick, but `update*` prefix for these methods now makes more sense, since they don't accept any arguments and so we aren't explicitly setting anything